apps/dapp/src/components/Pages/Ascend/components/LBPForm/index.tsx

```diff
@@ -119,73 +86,133 @@ export const LBPForm = ({ pool }: Props) => {
   };
 
   const createChangeHandler = (fieldKey: keyof Values) => (event: React.ChangeEvent<any>) => {
-    setFormValue(fieldKey, event.target.value);
+    let value = event.target.value;
+
+    if (['startWeight1', 'startWeight2', 'endWeight1', 'endWeight2'].indexOf(fieldKey) >= 0) {
```

Comment:
prob more appropriate to just do 
`['startWeight1', 'startWeight2', 'endWeight1', 'endWeight2'].includes(fieldKey)`

Reply:
https://github.com/TempleDAO/temple/pull/461/commits/bb507608688ba733e68f2319522ec1215bc9784f